setup.cfg

```diff
@@ -42,6 +42,18 @@ ignore = E126,E133,E226,E241,E242,E302,E704,E731,E722,W503,E402,W504,F821,E203
 exclude = .asv*,build/*,.tox/*,dev*/*,env/*,test_data/*,tests/*,ve/*,utils/*,*/_vendor/*,conda/compat.py,conda/common/compat.py,conda_env/compat.py
 
 
+[coverage:run]
+# store relative paths in coverage information
+relative_files = True
```

Comment:
This is an attempt to run coverage in and out of docker, and combine the results. It doesn't work perfectly but it is better than nothing.